.github/workflows/check-spelling.yml

```diff
@@ -27,18 +23,31 @@ on:
           cspell:npm/npm.txt
           cspell:dotnet/dotnet.txt
           cspell:scala/scala.txt
+      config:
+        description: Spelling configuration directory
+        required: false
+        type: string
+        default: .github/actions/spelling
+      spell-check-this:
+        description: Repository with default configuration to use
+        required: false
+        type: string
+        default: ''
```

Comment:
Have a default for this repo here, instead of having to specify this parameter in every consumer.